conductor/lib/file_utils.py

```diff
@@ -157,6 +157,13 @@ def process_upload_filepath(path, strict=True):
     paths = []
 
     if path:
+        try:
```

Comment:
Can you update the docstring to add this verification step?

Reply:
Thanks. Done.